libeos-updater-util/config.c

```diff
@@ -370,7 +370,7 @@ euu_config_file_get_file_for_key (EuuConfigFile  *self,
     }
 
   /* Not found? */
-  if (i >= self->n_paths)
+  if (i > self->n_paths)
```

Comment:
This makes sense in light of line 360 above

Reply:
Yes. I see what it does too. However, the loop in line 360 does look confusing and unsurprisingly leads to errors like what's being fixed here.
I will approve the changes too but I'd rather just use a loop where `i < n_paths` and abstract the loop's body logic in a different function in order to call it after the loop with `default_path` as an argument. I think this would make things clear, require no comments for understanding what's going on, and thus be less error prone.

Either way, it's @pwithnall 's choice.

Reply:
I don’t think abstracting the loop body out would particularly help with understandability. I think the comment is enough, and I don’t want to spend any more time on this at the moment. Thanks for the reviews!